src/main/java/com/bbd/modisa/CheckAccount.java

```diff
@@ -1,17 +1,52 @@
 package com.bbd.modisa;
 
+import java.util.Scanner;
+
```

Comment:
create an interface and have specific implementation for each account type. do not bundle the business logic with models.